app/server.go

```diff
@@ -486,7 +486,7 @@ func (s *Server) Start() error {
 	}
 	s.ListenAddr = listener.Addr().(*net.TCPAddr)
 
-	mlog.Info(fmt.Sprintf("Server is listening on %v", listener.Addr().String()))
+	mlog.Info("Server is listening ", mlog.String("on", listener.Addr().String()))
```

Comment:
I don't think this change is needed, it doesn't improve logging

Reply:
@reflog Thanks for the feedback. I'll revert the change.